RecoTracker/TkDetLayers/test/TkDetLayersAnalyzer.cc

```diff
@@ -116,8 +117,9 @@ TkDetLayersAnalyzer::analyze( const Event& iEvent, const EventSetup& iSetup )
     << " And Contains  Daughters: "<< (*pDD).components().size() ;   
 
   ESHandle<TrackerTopology> tTopo;
-  iSetup.get<IdealGeometryRecord>().get(tTopo);
+  iSetup.get<TrackerTopologyRcd>().get(tTopo);
 
+  /*
```

Comment:
why not remove this code completely?
If it's needed, better put a define/ifdef  and keep it alive.